k9mail-library/src/main/java/com/fsck/k9/mail/internet/FlowedMessageUtils.java

```diff
@@ -0,0 +1,20 @@
+package com.fsck.k9.mail.internet;
+
+
+import static com.fsck.k9.mail.internet.MimeUtility.getHeaderParameter;
+import static com.fsck.k9.mail.internet.MimeUtility.isFormatFlowed;
+
+
+public class FlowedMessageUtils {
```

Comment:
there is a FlowedMessageUtils class already that has the actual flowing/deflowing methods in it, so this name can be confusing. The method from this class should also be next to the `isFormatFlowed` one itself. I think we can just move it into `MimeUtility`, or name this differently and move isFormaFflowed in here. @cketti?